# username: 'test'
# password: '123456'
# broker_address: 60.28.24.166

username: 'tju_me'
password: 'tjuME!@#'
broker_address: 60.28.24.166